install/cluster-operator/040-Crd-kafka.yaml

```diff
@@ -174,7 +174,9 @@ spec:
                             enableECDSA:
                               type: boolean
                               description: Enable or disable ECDSA support by installing
-                                BouncyCastle crypto provider. Default value is `false`.
+                                BouncyCastle crypto provider. ECDSA support is always enabled.
```

Comment:
This file should not be changed - this is the CRD from 0.23 (all changes should be done to `packaging/...`). Can you please revert it?

Reply:
Reverted.